src/main/java/org/httpserver/config/ConfigurationManager.java

```diff
@@ -0,0 +1,66 @@
+package org.httpserver.config;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.httpserver.util.Json;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class ConfigurationManager {
+    private static ConfigurationManager myConfigurationManager;
+    private static Configuration myCurrentConfiguration;
+    private ConfigurationManager() {
+    }
+    public static ConfigurationManager getInstance(){
+        if (myConfigurationManager == null)
+            myConfigurationManager = new ConfigurationManager();
+        return myConfigurationManager;
+    }
+
+    /**
+     *
+     * Used to load a configuration file by the path provided
+     */
+    public void loadConfirurationFile(String filePath) {
```

Comment:
This method might be `public static` and return an instance of `Configuration`. 
And if so, method name should be changed in order to correctly reflect it's intentions.